apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'Enhancing Developer Experience at SciPy - Intel oneAPI/MSVC Support and Migrating to spin'
+published: February 21, 2025
+authors: [gagandeep-singh]
+description: 'Highlights the work done to improve developer experience at SciPy, specifically on supporting Intel oneAPI/MSVC and spin'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_featured.png
+  alt: 'SciPy logo'
+hero:
+  imageSrc: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_hero.png
+  imageAlt: 'SciPy logo'
+---
+
+In this blog post, we’ll discuss the improvements we made to enhance the developer experience at SciPy. Our work focused on two key areas:
+
+1. Intel oneAPI and MSVC Support in SciPy
+2. Transitioning from `dev.py` to `spin`
+
+First, let’s explore the motivation behind these updates.
+
+Intel provides a comprehensive toolkit for high-performance computing (HPC) and numerical computing use cases, known as [oneAPI](https://www.intel.com/content/www/us/en/developer/tools/oneapi/overview.html).
+Given that SciPy is heavily used in HPC and numerical computing, supporting the ability to build SciPy with Intel oneAPI was a natural and important step.
+By doing so, we aim to improve the experience for SciPy users who rely on Intel oneAPI as their primary toolkit.
+Additionally, since MSVC is the de facto compiler for C/C++ on Windows, we added continuous integration (CI) jobs to test SciPy compilation with MSVC, further expanding the compatibility of SciPy across platforms.
+
+Now, let’s talk about the transition from `dev.py` to `spin`.
+`dev.py` (also known as `do.py` [[1]](https://labs.quansight.org/blog/the-evolution-of-the-scipy-developer-cli)) was a significant step forward in improving the development workflow.
```

Comment:
```suggestion
`dev.py` (also known as `do.py`, learn more in [The evolution of the SciPy developer CLI](https://labs.quansight.org/blog/the-evolution-of-the-scipy-developer-cli)) was a significant step forward in improving the development workflow.
```

We use this paradigm for footnotes. For links, having descriptive text is more accessible.